File: 1550-Three-Consecutive-Odds/main.go
package main

func threeConsecutiveOdds(arr []int) bool {

	n := len(arr)

	count := 0
	for i := 0; i < n; i++ {
		if arr[i]&1 != 0 {
			count++
		} else {
			count = 0
		}

		if count == 3 {
			return true
		}
	}

	return false
}

File: 1550-three-consecutive-odds/main.py
from typing import List

class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        
        n = len(arr)

        count = 0
        for i in range(n):

            if arr[i] & 1 != 0:
                count+=1
            else:
                count = 0

            if count == 3:
                return True
        

        return False
        